### PR TITLE
Change Skin -> Theme

### LINKED ISSUE #5: Recommended Name Changes
Based on the terms used in other platforms, I would suggest a rename of the Skins concept to be Theme.  This more closely aligns to other tools, which will help with understanding/adoptions.

I would gladly submit a PR for this, but would want to be sure it was desired before I do

### PR DESCRIPTION
**Pull request by mitchelsellers**
This resolves #5.

Renamed the concept of a "Skin" to be the concept of a "Theme".  This was done globally across all project references and has been tested to be working

## Limitation on Implementation ##

Given the current state of this project, this PR only addresses initial setup.  As it wasn't 100% clear of the best way to include a Migration for schema changes necessary for updating.

### CODE REVIEW

**Comment by sbwalker** [comment]
The repository home page states "At this point in time we do not promise any upgrade path from one version to the next, and developers should expect breaking changes as the framework stabilizes." - so no worries about the lack of a migration for schema changes.

File: Oqtane.Server/appsettings.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Oqtane.mdf;Initial Catalog=Oqtane;Integrated Security=SSPI;"
+    "DefaultConnection": "Data Source=(LocalDb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Oqtane-201905122226.mdf;Initial Catalog=Oqtane-201905122226;Integrated Security=SSPI;"
```
Comment (sbwalker) [inline Oqtane.Server/appsettings.json]: This file should not be included in the pull request. When the application is run for the first time, it modifies the appsettings.json to include a unique name for the LocalDB database so that it does not conflict with any existing database on a users machine.

Comment (mitchelsellers) [reply Oqtane.Server/appsettings.json]: Fixed that one

**Comment by mitchelsellers** [comment]
Updated the PR to remove the file

**Comment by nvisionative** [comment]
My .gitignore PR should have made this file untracked.

**Comment by sbwalker** [comment]
I will accept this pull request as soon as I can. In the process of converting the @oqtane personal account to an organization account something went wrong with the GitHub permissions and my owner account @sbwalker only has read privileges. I have contacted GitHub support about this issue.

**Comment by sbwalker** [comment]
https://github.community/t5/How-to-use-Git-and-GitHub/BUG-Converted-Personal-Account-to-Organization-but-Owner-only/td-p/23901

**Comment by nvisionative** [comment]
@sbwalker this is unfortunately why @nvisionative is not an "organization" account.  After reading all the _fine print_ for the conversion process, I decided to keep it as a "personal" account.  It was created prior to the existence of the "organization account" concept, so essentially we're stuck.  I hope you are able to figure this out with GitHub support.  😦 Good luck!  🍀

**Comment by sbwalker** [comment]
Received an email from GitHub Support this morning "Thanks for contacting GitHub Support! We're really sorry it's taken so long to respond to you. We are experiencing a very high volume of support requests at the moment, and it's causing us some delays. Yes, this is indeed a known bug - I'll get the engineering team to sort you out ASAP!"

**Comment by sbwalker** [comment]
Thank you Mitch for your patience!!

**Comment by sbwalker** [comment]
Note - this affects a few database tables and since we do not yet support database migrations, you will need to create a new database once you pull down the latest code. The easiest way to do this is to open the appsettings.json file in the Oqtane.Server project and revert the connection string to the default setting:

"Data Source=(LocalDb)\\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\\Oqtane.mdf;Initial Catalog=Oqtane;Integrated Security=SSPI;"

After you build and rerun the application it will create a new LocalDB, update your connection string, and provision the new schema and data.

**Comment by mitchelsellers** [comment]
Thanks Shaun! 

I'll try to get the follow-on change that I logged an issue on in over the weekend
